test/test_logging.py

```diff
@@ -110,22 +110,19 @@ def test_logging_to_default_handler(self):
     def test_logging_to_file(self):
```

Comment:
I don't quite understand why the changes to this method make the test which gets executed after this one (namely `test_logging_to_handler`) fail..

@manoelmarques could you please take a look at this? I don't seem to find the reason for this cross-test dependence

Reply:
Fixed the unit test method